src/components/reference-editor/suggestions/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { fromJS } from 'immutable';
+import EntryComponent from './entry';
+
+export default function (MentionSuggestions) {
+  return ({ suggestions, ...rest }) => (
+    <MentionSuggestions
```

Comment:
Not really, looks like there isn't a lot of styling in `List` itself and `ListItems` are just enough